packages/plugin-google-analytics/lib/index.ts

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-unused-vars, class-methods-use-this */
+import {
+  Event, Properties, RequestLoggerPlugin, PluginLoadOptions, PageOptions, IdentifyOptions, TrackOptions,
+} from '@itly/sdk';
+
+type GAOptions = {
+  gtag?: Gtag.Gtag;
+  measurementId: string;
+}
+
+export class GoogleAnalyticsPlugin extends RequestLoggerPlugin {
+  private _gtag?: Gtag.Gtag;
+
+  get gtag(): Gtag.Gtag {
+    this._gtag = this._gtag || self.gtag;
+    if (!this._gtag) throw new Error('No gtag provided to constructor, or available on window');
+    return this._gtag;
+  }
+
+  constructor(private opts: GAOptions) {
+    super('google-analytics');
+    if (this.opts.gtag) {
+      this._gtag = this.opts.gtag;
+    } else if ('gtag' in self) {
+      this._gtag = self.gtag;
+    } else {
+      this.initializeGtag(this.opts.measurementId);
```

Comment:
In the other Plugins we setup/initialize the trackers in load() instead of the constructor.